### ISSUE TITLE
Output format

### ISSUE THREAD

**Issue by dndeus** [sj-i/php-profiler#39]
Hello there

Is there a way to view the results of the profile as a flamegraph r something?? If so can you provide with an example?

Thank you for your efforts

**Comment by sj-i** [comment]
Hi! Sorry for the late reply.
I'm slowly working on a generalized solution for customizing the output format, but I don't have much time to do it for now.
Just adding ad hoc support for outputting a flame graph is easy. So I'll give it a shot in days.

**Comment by sj-i** [comment]
@dndeus
I've just released v0.2.1. This release includes tools for generating flamegraphs borrowed from phpspy. The default format of the traces is adjusted at v0.2.0 to match the output of phpspy, so you can now generate flamegraphs by using commands like following.
```bash
sudo php ./php-profiler inspector:trace -p [pid of the target process or thread] >traces
./tools/stackcollapse-phpspy/stackcollapse-phpspy.pl <traces | ./tools/flamegraph/flamegraph.pl >flame.svg
```
